src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -254,7 +254,7 @@ private ZipFileProcessingResult processZipFileContent(
 
             InputEnvelope envelope = envelopeProcessor.parseEnvelope(result.getMetadata(), zipFilename);
 
-            EnvelopeValidator.assertContainerMatchesJurisdiction(envelope, containerName);
+            envelopeProcessor.assertContainerMatchesJurisdictionAndPoBox(envelope.jurisdiction, envelope.poBox, containerName);
```

Comment:
why not keep it in the `EnvelopeValidator`?

Reply:
I tried to load ConfigurationProperties in `EnvelopeValidator` but it's not working. So I moved it to `EnvelopeProcessor`

Reply:
maybe the configuration can be passed as a parameter then.

Reply:
yes it can be passed, and I tried that. But the configuration properties will not be loaded if the `BlobProcessorTask` scheduling is turned off. For integration tests, it is turned off

Reply:
I moved the validation method to `EnvelopeValidator`. 
I guess I was missing something earlier when I tried to pass the configuration from `BlobProcessorTask`.